web/app.js

```diff
@@ -1816,7 +1816,7 @@ const PDFViewerApplication = {
     this.pdfViewer.cleanup();
     this.pdfThumbnailViewer?.cleanup();
 
-    this.pdfDocument.cleanup();
+    this.pdfDocument.cleanup(/* keepLoadedFonts = */ AppOptions.get("pdfBug"));
```

Comment:
I'm not sure that I understand, more specifically, why this is necessary for *non*-embedded fonts? I.e. what potentially "breaks" in the `Font Inspector` unless we keep fonts around?

But, assuming that this is in fact necessary:
```suggestion
    this.pdfDocument.cleanup(/* keepLoadedFonts = */ AppOptions.get("fontExtraProperties"));
```

Reply:
In the devtools, in the font panel, we've some infos about the font used. If we clear the cache then we'll see the font used as ultimate fallback (serif/sans serif).